python/lsst/ts/xml/generate_sal_generics_doc.py

```diff
@@ -0,0 +1,89 @@
+# This file is part of ts_xml
+#
+# Developed for the LSST Data Management System.
```

Comment:
Please change this line to `Developed for the Vera Rubin Observatory Telescope and Site Systems.`.